tinkernet/runtime/src/xcm_config.rs

```diff
@@ -21,8 +21,11 @@ use orml_traits::{
 pub use orml_xcm_support::{
     DepositToAlternative, IsNativeConcrete, MultiCurrencyAdapter, MultiNativeAsset,
 };
+use pallet_ocif_staking::primitives::{
+    CustomAggregateMessageOrigin, CustomMessageProcessor, CustomNarrowOriginToSibling,
+    CustomParaIdToSibling,
+};
```

Comment:
Why is there XCM stuff in the staking pallet?

Reply:
abstractions required to not interfere with the current message queue functionality when adding a different MessageProcessor

Reply:
In the future please move this to the runtime as it's not related to the staking pallet.